CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ endif()
 
 set(CMAKE_AUTOMOC ON)
 
-OPTION(WITH_QT6 "Enable Qt 6" OFF)
+OPTION(WITH_QT6 "Enable Qt 6" ON)
```

Comment:
This changes using Qt 6 by default everywhere and would break a couple of builds for Linux distros.

Would it be possible to make this automatic? Specifically:
- By default if the option is not overridden, pick Qt 6 preferably and if not found fallback to Qt 5.
- If set to ON by user, require Qt 6.
- If set to OFF by user, require Qt 5.